ui/libproject/devstart.rb

```diff
@@ -0,0 +1,86 @@
+require "optparse"
+require_relative "utils/common"
+
+def install_dependencies()
+  common = Common.new
+  common.docker.requires_docker
+
+  common.run_inline %W{docker-compose run --rm ui npm install}
+end
+
+def swagger_regen()
+  common = Common.new
+  common.docker.requires_docker
+
+  common.run_inline %W{docker-compose run --rm ui npm run codegen}
+end
+
+class DevUpOptions
+  ENV_CHOICES = %W{local test prod}
+  attr_accessor :env
+
+  def initialize
+    self.env = "test"
+  end
+
+  def parse args
+    parser = OptionParser.new do |parser|
+      parser.banner = "Usage: ./project.rb dev-up [options]"
+      parser.on(
+          "--environment ENV", ENV_CHOICES, "Environment [local (default), test, prod]") do |v|
```

Comment:
I believe that this will only take `--environment`, rather than some other npm flag names. (Like `-e` or `--env`)
I don't think its a big problem, but wanted to bring it up.